### ISSUE TITLE
QuillField parse error if null

### ISSUE THREAD

**Issue by michaldyczko** [LeeHanYeong/django-quill-editor#16]
If QuillField is not populated, it causes JSONParse error. It can be field by overriding `FieldQuill.__init__` with:
```
def __init__(self, instance, field, json_string):
    json_string = json_string or '{"delta":"","html":""}'
    super().__init__(instance, field, json_string)
```
After that, QuillField blank and null arguments can be set to True.

**Comment by LeeHanYeong** [comment]
Thanks for the error report. Would you please send a PR if you have time? 
If you aren't familiar with PR, I'll update the new commit I mentioned you.

**Comment by michaldyczko** [comment]
Hi Lee, I will make a PR in a free time :)

**Comment by LeeHanYeong** [comment]
Merged c6b287739bbbf34105ceaf797e6fd1538082f0d4
